frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -56,12 +58,21 @@ export const UnsubscribeSurveyModal = ({
             }
         >
             <div className="flex flex-col gap-3.5">
-                {unsubscribeError && (
+                {unsubscribeError ? (
                     <LemonBanner type="error">
                         <p>
                             {unsubscribeError.detail} {unsubscribeError.link}
                         </p>
                     </LemonBanner>
+                ) : (
+                    <LemonBanner type="warning">
+                        <p>
+                            We will attempt to pay any open invoices associated with your account when you unsubscribe.{' '}
+                            <Link to={billing?.stripe_portal_url} target="_blank">
+                                View invoices
+                            </Link>
```

Comment:
Does this show them their upcoming bill?

Reply:
Their upcoming bill won't exist until after they unsubscribe. This links to the customer portal, where they can see their invoice history. I can, alternatively, return the link to the hosted portal for the first open invoice we find.